docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -118,16 +130,32 @@ While app schemes are generally easier to set up, Universal URLs and App Links a
   </mark>
 </p>
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 Calling the token endpoint is the second step in the authorization flow and usually happens in your backend if your application's Client Type is `Confidential` <mark>What if the Client Type is **Public**?</mark>. Make sure to a call the authorization endpoint first to retrieve an authorization code.
```

Comment:
Good question. @ismailmustafa, what does change here? Does that just change the request parameters?